JitsiConference.ts

```diff
@@ -2641,7 +2653,7 @@ export default class JitsiConference extends Listenable {
     /**
      * Handles the suspend detected event. Leaves the room and fires suspended.
      * @param {JingleSessionPC} jingleSession - The Jingle session.
-     * @private
+     * @internal
```

Comment:
This on the contrary is part of the public API, Jitsi Meet calls this method. The documentation is not up to date looks like.